fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMgr.java

```diff
@@ -2,34 +2,69 @@
 
 package com.starrocks.connector;
 
-import java.util.Map;
+import com.google.common.base.Preconditions;
+import com.starrocks.common.DdlException;
+import com.starrocks.connector.hive.HiveConnectorFactory;
+import com.starrocks.server.MetadataMgr;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
 
 // ConnectorMgr is responsible for managing all ConnectorFactory, and for creating Connector
 public class ConnectorMgr {
-    private final ConcurrentMap<String, ConnectorFactory> connectorFactories = new ConcurrentHashMap<>();
-
-    /**
-     * add connectorFactory
```

Comment:
I will add comments at the end. because the parameters keep changing. it is easy to expire